src/schema.py

```
@@ -1162,6 +1175,23 @@ def mutate(self, info, user_id):
         user = User.get_query(info).filter(UserModel.id == user_id).first()
         if not user:
             raise GraphQLError("User with given ID does not exist.")
+        
+        s3 = boto3.client('s3',
+            endpoint_url=os.getenv("DIGITAL_OCEAN_URL"),
```

Flake8 (7.3.0)

[error] continuation line under-indented for visual indent

(E128)